### ISSUE TITLE
Staging:   TMP VVC Personal Device Appointment Missing Detailed Appt Data

### ISSUE THREAD

**Issue by marcynadeau** [department-of-veterans-affairs/va.gov-team#40970]
TMP - Telehealth Management Portfolio
VVC - VA Video Connect

**Steps to Recreate** 
TMP Steps
1. Magnifying glass select Ralph Marshall
2. Contact Information  - Technology Type = Personal device.
3. Select Schedule Appt in top banner
4. Appt Modality = VVC
5. Provide Site = Middletown, OH CBOC (984GA)
6. Provider = Dubbaka, Naveen
7. Scheduling Package = Test Service CVT - Pat VVC, Pro CD, Ind, Interfacility VAOS VVC
8. Log into 984.  Validate three VistA  logins show three green checks. 
9. From top banner use Find available times. 
10. Then in side window set Requested Date only select ASAP.  Then select find available times in side window.   
11. Select a MID-V10 CRH MH PSY MD VVC resource with date/time.  Select Schedule.  
12. From top banner select Save. 
13. In middle of screen select Appointment Information to see the booked appointment

VAOS Steps

1. Log into VAOS as ralph.marshall@Id.me.  View the upcoming appointment list.  Find the TMP Appointment.  
2. Expected:  VAOS shows appointment as VA Video Connect at Home. VAOS shows similar instructions as seen when on VAR Resources.  
3.  Actual: VAOS is missing instructions for joining the call and the link within the app to join 30 minutes before is missing.  The name of the provider is also missing (You'll be meeting with).  

This is the endpoint that returns the data in VAOS Service screen shot:  https://staging-api.va.gov/vaos/v2/appointments?_include=facilities,clinics&start=2022-04-05&end=2023-06-04&statuses[]=booked&statuses[]=arrived&statuses[]=fulfilled&statuses[]=cancelled

**Comment by marcynadeau** [comment]
![RalphVAOS_v_VarRes_VVC_PersonalDevice.png](https://images.zenhubusercontent.com/5f2d82e4a65f7435058cfd75/06eb583b-a1a7-4e00-a9f5-15874193927d)

**Comment by ldelacosta** [comment]
Re-assigning back to the dev team.

**Comment by marcynadeau** [comment]
@ldelacosta @JRRoof This was discussed on a call with Front End and back end developers.  It was determined this is a front end issue. 
For a future enhancement Brad Crosby will look into  'beefing up the API' so that the client does not have to infer the modality.

**Comment by ldelacosta** [comment]
Just an update that VAOSR-3701 is completed.

**Comment by vbahinwillit** [comment]
![staging va gov_health-care_schedule-view-va-appointments_appointments_ (1)](https://user-images.githubusercontent.com/3587066/171699061-82a20416-be18-4e09-9c02-4a370136c56c.png)

![staging va gov_health-care_schedule-view-va-appointments_appointments_](https://user-images.githubusercontent.com/3587066/171698943-eb87cd50-f2f7-4d13-82ae-b87e1be4f067.png)

```
{
    "id": "99174",
    "type": "appointments",
    "attributes": {
        "id": "99174",
        "identifier": [
            {
                "system": "http://med.va.gov/fhir/urn/vaos/vista/984/appointment/id",
                "value": "4290;20220613.120000"
            },
            {
                "system": "http://med.va.gov/fhir/urn/vaos/vvs/id",
                "value": "ef7d0d8e-dbcb-ec11-983e-001dd8031ddb"
            }
        ],
        "kind": "telehealth",
        "status": "booked",
        "patientIcn": "1012830757V803997",
        "locationId": "984",
        "clinic": "4290",
        "practitioners": [
            {
                "name": {
                    "family": "Adkins",
                    "given": [
                        "Edna"
                    ]
                },
                "practiceName": "DAYTSHR"
            }
        ],
        "start": "2022-06-13T16:00:00Z",
        "end": "2022-06-13T16:30:00Z",
        "minutesDuration": 30,
        "slot": {
            "id": "3230323230363133313630303A323032323036313331363330",
            "start": "2022-06-13T16:00:00Z",
            "end": "2022-06-13T16:30:00Z"
        },
        "cancellable": false,
        "telehealth": {
            "url": "https://care2.evn.va.gov/vvc-app/?join=1&media=1&escalate=1&conference=TMPX0305143@care2.evn.va.gov&pin=8112506403#",
            "group": false,
            "vvsKind": "MOBILE_ANY"
        },
        "extension": {
            "ccLocation": {
                "address": {}
            },
            "vistaStatus": [
                "CANCELLED BY CLINIC",
                "CANCELLED BY PATIENT",
                "NO ACTION TAKEN",
                "FUTURE"
            ]
        },
        "location": {
            "id": "984",
            "type": "appointments",
            "attributes": {
                "id": "984",
                "vistaSite": "984",
                "vastParent": "984",
                "type": "va_health_facility",
                "name": "Dayton VA Medical Center",
                "classification": "VA Medical Center (VAMC)",
                "timezone": {
                    "timeZoneId": "America/New_York"
                },
                "lat": 39.74935,
                "long": -84.2532,
                "website": "https://www.dayton.va.gov/locations/directions.asp",
                "phone": {
                    "main": "937-268-6511"
                },
                "physicalAddress": {
                    "type": "physical",
                    "line": [
                        "4100 West Third Street"
                    ],
                    "city": "Dayton",
                    "state": "OH",
                    "postalCode": "45428-9000"
                },
                "healthService": [
                    "Audiology",
                    "Cardiology",
                    "DentalServices",
                    "Dermatology",
                    "Gastroenterology",
                    "Gynecology",
                    "MentalHealthCare",
                    "Nutrition",
                    "Ophthalmology",
                    "Optometry",
                    "Orthopedics",
                    "Podiatry",
                    "PrimaryCare",
                    "SpecialtyCare",
                    "Urology",
                    "WomensHealth"
                ]
            }
        }
    }
}
```

**Comment by ldelacosta** [comment]
@marcynadeau - Please validate and let us know if you have any questions. Thanks!

**Comment by marcynadeau** [comment]
Testing on hold. Ran into an issue with TMP and reached out to TMP team for help.

**Comment by marcynadeau** [comment]
@ldelacosta I am blocked on testing this defect. Something is not working as expected in the TMP test account and I need help from their dev. I think it is a configuration issue within TMP that needs to be fixed.  I have reached out to Mario on TMP for assistance.

**Comment by marcynadeau** [comment]
Tested and passed testing. 
 ![VVC_PersonalDevice_06212022.png](https://images.zenhubusercontent.com/5f2d82e4a65f7435058cfd75/2a7fc690-1899-4f56-a7f9-9af25c2cb2f8
)
